common/Root.tsx

```diff
@@ -262,6 +238,29 @@ const LegacyRoutes = withRouter(props => {
   );
 });
 
+const DevModeToggle = () => {
+  const { developmentMode, toggleDevMode } = useDevMode();
```

Comment:
For boolean flags please follow `is/has + noun` convention
https://www.samanthaming.com/tidbits/34-better-boolean-variable-names

It helps the developer to know immediately that he is dealing with a boolean, and he knows which way to set it

Reply:
I'll keep this in mind. It definitely makes sense!